Source: Bo Andresen & Ingmar Vanhassel
Upstream: Nope
Reason: Fix library versioning to match the versions from Qt 4.5.0

From 6a16e71118b27c9af098e50b95cd9d5ea4af70c1 Mon Sep 17 00:00:00 2001
From: Ingmar Vanhassel <ingmar@exherbo.org>
Date: Fri, 6 Mar 2009 10:11:28 +0100
Subject: [PATCH] Fix version for 4.3.1 release.

---
 CMakeLists.txt |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index c200157..c18c3e7 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -149,8 +149,8 @@ set(CMAKE_COLOR_MAKEFILE ON)
 
 set(PHONON_LIB_MAJOR_VERSION "4")
 set(PHONON_LIB_MINOR_VERSION "3")
-set(PHONON_LIB_PATCH_VERSION "0")
-set(PHONON_LIB_VERSION "${PHONON_LIB_MAJOR_VERSION}.3.0")
+set(PHONON_LIB_PATCH_VERSION "1")
+set(PHONON_LIB_VERSION "${PHONON_LIB_MAJOR_VERSION}.3.1")
 set(PHONON_LIB_SOVERSION ${PHONON_LIB_MAJOR_VERSION})
 
 add_definitions(${QT_DEFINITIONS})
-- 
1.6.2

